prisma/migrations/20240803171237_setup_schema/migration.sql

```diff
@@ -0,0 +1,270 @@
+-- CreateEnum
```

Comment:
When creating migrations, 1 file should focus on 1 entity, e.g. `User`, `Course`, etc. Migration file name should clearly indicate what the query is doing.